tests/test_renderer.py

```diff
@@ -1,10 +1,25 @@
 import pytest
 
 import pyvista
+from pyvista import _vtk
 from pyvista.plotting import system_supports_plotting
 from pyvista.plotting.renderer import ACTOR_LOC_MAP
 
 
+def test_show_bounds():
+    plotter = pyvista.Plotter()
+    bounds = (0.0, 1.0, 0.0, 1.0, 0.0, 1.0)  # Renderer bounds
+    axes_range = (0.0, 5.0, 0.0, 5.0, 0.0, 5.0)  # Axes bounds
+    plotter.show_bounds(bounds=bounds, axes_range=axes_range)
+    cube_axes_actor = plotter.renderer.cube_axes_actor  # Pull the actor
+    assert isinstance(cube_axes_actor, _vtk.vtkCubeAxesActor)  # Check type
+    assert cube_axes_actor.GetBounds() == bounds  # Check the bounds
+    # Check the axes ranges
+    assert cube_axes_actor.GetXAxisRange() == axes_range[0:2]
+    assert cube_axes_actor.GetYAxisRange() == axes_range[2:4]
+    assert cube_axes_actor.GetZAxisRange() == axes_range[4:]
```

Comment:
Looking at this I realise that my suggestion to expose the `axes_range` as a property wouldn't easily work, because it's an attribute that lives on the cube axes actor. I had `bounds` in mind, but due to the padding even the bounds of the actor could be different from the bounds of the mesh. It might not be worth trying to expose that at the renderer level (assuming it would even make sense to define a single bounds and axes range for renderers).

Reply:
See above for the added plotter property suggestion.